src/Uno.UWP/UI/Core/CoreWindow.cs

```diff
@@ -6,9 +6,17 @@
 
 namespace Windows.UI.Core
 {
-	public partial class CoreWindow 
+	public partial class CoreWindow
 	{
-		public global::Windows.UI.Core.CoreDispatcher Dispatcher
+		public CoreDispatcher Dispatcher
 			=> CoreDispatcher.Main;
+
+		[Uno.NotImplemented]
+		public CoreVirtualKeyStates GetAsyncKeyState(System.VirtualKey virtualKey)
```

Comment:
Seems unrelated to the pr

Reply:
Actually this PR contains multiple minor changes made to support the SyncFusion DataGrid, however in comment I focused on the most important changes instead of writing useless comment like :
## Current Behavior
`VirtualKeyStates` throw exceptin

## New behavior
`VirtualKeyStates` always return `None`

Reply:
Splitted !